src/Components/Patient/PatientRegister.tsx

```diff
@@ -375,7 +375,7 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             village: res.data.village ? res.data.village : "",
             medical_history: [],
             is_antenatal: res.data.is_antenatal
-              ? res.data.is_antenatal
+              ? String(res.data.is_antenatal)
```

Comment:
Use `is_antenatal: String(!!res.data.is_antenatal)`. It looks more clean and readable.

Reply:
Hey, thanks for the suggestion. Would you recommend me changing all other occurrences of such typecasting too (In this file)? 
![image](https://user-images.githubusercontent.com/3626859/180713334-7c5b827b-ebdb-4bc7-bc58-e6883c8cb54b.png)
I used `String(` to maintain consistency with the existing codebase but would be happy to change it everywhere in the file, as per your suggestion, if you believe that's the better approach.